rhel/usr_lib_systemd_system_ovn-ic.service

```diff
@@ -23,7 +23,6 @@ RemainAfterExit=yes
 Environment=OVN_RUNDIR=%t/ovn OVN_DBDIR=/var/lib/ovn
 EnvironmentFile=-/etc/sysconfig/ovn
 EnvironmentFile=-/etc/sysconfig/ovn-ic
-ExecStartPre=-/usr/bin/chown -R ${OVN_USER_ID} ${OVN_DBDIR}
```

Comment:
Why is this removed ?

Reply:
This command is useless in case we ovn-ic.service is only responsible for running ovn-ic daemon. Since we run ovsdb-servers for IC in separate systemd unit, we'll chown OVN_DBDIR there.

Reply:
Makes sense. Thanks for answering the question.